README.md

```diff
@@ -1,10 +1,10 @@
-# csde-components-validator
+# studio-component-set-tools
 
-Validation module for Content Station Digital Editor component sets.
+Tools module for Studio Digital Editor component sets.
 
 ## Usage
 
-The module provides two public api methods for validating a component set.
+The module provides tooling to develop component sets. It contains two public api methods for validating a component set.
```

Comment:
It actually contains three public validation methods 😇  Might be good to rephrase this a bit with mentioning the number of methods.

Reply:
Updated it, please have another look.

Reply:
`It contains for example...` -> `For example, it contains...`?

Reply:
That is better indeed.